docs/modules/ROOT/pages/get-started.adoc

```diff
@@ -1,9 +1,246 @@
+hazelcast:
+  cluster:
+    name: "<DEV_CLUSTER_NAME>"
+    cloud:
+      token: "<DEV_DISCOVERY_TOKEN>"
+      enabled: true
+----
++
+NOTE: Different parameter values are required when connecting to clusters on Hazelcast Platform. See xref:configuration.adoc[] for full details.
+. Save your changes.
+
+[[step-2-dev-connect]]
+== Step 2. Connect the CLC to the Development Cluster
+You're ready to connect the CLC to your development cluster. Make sure that the cluster is running before you start.
+
+. From the command line, change into `$HOME/.local/share/hz-cli`.
+. Execute the following command:
++
+```bash
+hzc
+```
++
+The CLC connects to the development cluster and the command prompt displays the cluster details. For example: `hzc hazelcast-cloud@pr-3145>`.
+. Press kbd:[Ctrl+C] or type 'exit' to shut down the CLC while you complete the configuration for your production cluster.
+
+[[step-3-prod-configure]]
+== Step 3. Add Configuration for the Production Cluster
+Now add the security keys and a new configuration file for your production cluster.
+
+. Save the security keys for the production cluster to a separate directory on your local machine. For example: `$HOME/.local/share/*prod*`.
+. Return to `$HOME/.local/share/hz-cli`, make a copy of `config.yaml` and save it to the same directory. Give the new file a unique name. For example, `config_prod.yaml`.
+. Open the new configuration file and update the following parameters with values for your production cluster.
++
+[source,yaml]
+----
+ssl:
+  enabled: true
+  capath: "/<PATH>/ca.pem"
+  certpath: "/<PATH>/cert.pem"
+  keypath: "/<PATH>/key.pem"
+  keypassword: "<PROD_KEYSTORE_PASSWORD>"
+hazelcast:
+  cluster:
+    name: "<PROD_CLUSTER_NAME>"
+    cloud:
+      token: "<PROD_DISCOVERY_TOKEN>"
+      enabled: true
+----
+. Save your changes.
+
+[[step-4-cluster-switch]]
+== Step 4. Switch Between Clusters
+Having separate local configurations for your development and production clusters allows you to switch between them without leaving the command line. Make sure both clusters are running before you start.
+
+. From the command line, change into `$HOME/.local/share/hz-cli`.
+. Execute the following command to use the default configuration to connect to your development cluster:
++
+```bash
+hzc
+```
++
+As before, the CLC connects to your development cluster.
+. Type `exit` to return to `$HOME/.local/share/hz-cli`.
+. Execute the following command to use the new configuration file for your production cluster.
++
+```bash
+hzc -c <config_prod>.yaml
+```
++
+The CLC connects to your production cluster
+
+[[step-5-write-data]]
+== Step 5. Write Data to a Map
+Now that you've connected to both your clusters, try using the CLC to write data to a map on your development cluster. A quick and easy way to do this is to use a JSON file.
+
+. Copy the following JSON to a file in the directory `$HOME/.local/share/hz-cli`.
++
+.currencycodes.json
+```json
+{
+  "1": {
```

Comment:
Maybe we should use the currency codes as the keys?

I'm not sure how this works but I assume the `put-all` command uses top-level JSON fields as the keys for the map?

Reply:
Yes, hoping I can test this at some point soon and make adjustments if necessary. This is the PR for the change: https://github.com/hazelcast/hazelcast-commandline-client/pull/118

Reply:
This works as expected, i.e. the top-level JSON fields are used as keys.